pkg/documentation/generator/description.go

```diff
@@ -73,8 +74,8 @@ func createDescriptionSection(stepData *config.StepData) string {
 		description += headlineGHA
 		description += fmt.Sprintf("%v```\n", spacingTabBox)
 		description += fmt.Sprintf("%vsteps:\n", spacingTabBox)
-		description += fmt.Sprintf("%v  - name: your preferred name for the step\n", spacingTabBox)
-		description += fmt.Sprintf("%v    uses: SAP/project-piper-action@main\n", spacingTabBox)
+		description += fmt.Sprintf("%v  - name: %v\n", spacingTabBox, stepData.Metadata.Name)
+		description += fmt.Sprintf("%v    uses: SAP/project-piper-action@releaseCommitSHA\n", spacingTabBox)
```

Comment:
Let's use the version here, I think that's more intuitive to use. 
```suggestion
		description += fmt.Sprintf("%v    uses: SAP/project-piper-action@1.23.0\n", spacingTabBox)
```

Would be interesting to know if we can use `@1` as well.